rohmu/transfer_pool.py

```diff
@@ -0,0 +1,220 @@
+"""
+rohmu - transfer_pool
```

Comment:
Implementation follows the robjecter one. Differences:
* Using a class instead of just globals
* Provide context-manager interface
* Use a proxy for the "get" interface to try and help finding bugs related to users forgetting the "return" call to the pool
* Optimizing the pool if backend is thread-safe
* minor refactorings